src/azure-cli-core/azure/cli/core/authentication.py

```diff
@@ -21,7 +21,10 @@ def _convert_token_entry(token):
 
 
 def _create_scopes(resource):
-    scope = resource.rstrip('/') + '/.default'
+    if resource == 'https://datalake.azure.net/':
```

Comment:
Does datalake endpoint differ in different cloud ?

Reply:
it only exists in public cloud now